### DISCUSSION TITLE
JuliaCall: How to convert list[str] to Vector{String}

### DISCUSSION THREAD

**Discussion by aaeberharter** [cjdoris/PythonCall.jl#354] [Q&A]
I am trying to call a Julia function from Python with Python objects as arguments. The function requires an argument of type Vector{String}, I tried the following:

```
from juliacall import Main as jl, convert as jlconvert

jl.seval('using MyLibrary')

x = ['a', 'b', 'c']
ret = my_function(keyword = jl.Vector(x))
```

Unfortunately it gives me the error
`juliacall.JuliaError: TypeError: in keyword argument types, expected Union{Nothing, Vector{String}}, got a value of type Vector{Any}`

How can I correctly perform this conversion?

**Comment by cjdoris** [comment]
Instead of `jl.Vector(x)` you can do `jlconvert(jl.Vector, x)` or `jlconvert(jl.Vector[jl.String], x)`.
